package.json

```diff
@@ -5,15 +5,16 @@
   "main": "dist/main.js",
   "scripts": {
     "clean": "rm -rf dist",
-    "start": "ulimit -n 999999 || ulimit -n 2048; npm run build && node dist/main.js -p 8080 --log-level=debug",
+    "start": "node dist/main.js -p 8080 --log-level=debug",
     "serve_local": "npm run build && nodemon dist/main.js -- --dev",
     "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js' --require @babel/register --require test_setup.js",
     "deploy": "./deploy.sh",
     "deploy:docker": "DOCKER=1 ./deploy.sh",
     "set_default": "./set_default.sh",
     "lint": "flow && eslint --config .eslintrc ./*.js src",
     "prettyquick": "pretty-quick",
-    "build": "npm run clean && babel src --out-dir dist --source-maps --ignore src/testdata",
+    "build:noclean": "babel src --out-dir dist --source-maps --ignore src/testdata",
```

Comment:
Interesting - is this command the one that's run on deploy now? I kind of feel like the noclean version should be the default and one with clean should be the alternative.

Reply:
No, it was my early attempt to fix, but the build itself also needs to not run on GAE as it wants to write files, and the GAE file system is read-only.

The build instead occurs during `npm install` as the `postinstall` step and then those built files are copied to GAE on deploy.